### PR TITLE
Modernizes poddoor.dm

### PR DESCRIPTION
**Pull request by SuperNovaa41**
## About The Pull Request
saw this very outdated code while hunting for bugs and it made me sick so i fixed it
if something looks wrong please tell me

## Changelog

code improvement

### CODE REVIEW

File: code/game/machinery/doors/poddoor.dm [hunk 1]
```diff
@@ -16,56 +15,70 @@
 	damage_deflection = 70
 	var/datum/crafting_recipe/recipe_type = /datum/crafting_recipe/blast_doors
 	var/deconstruction = BLASTDOOR_FINISHED // deconstruction step
+	var/id = 1
```
Comment (jjpark-kb) [inline code/game/machinery/doors/poddoor.dm:18]: since these are vars, dont they need /// ?

Comment (SuperNovaa41) [reply code/game/machinery/doors/poddoor.dm:18]: i didn't add this, also i think a comment is only really necessary for vars that _need_ explaining

File: code/game/machinery/doors/poddoor.dm [hunk 2]
```diff
@@ -16,56 +15,70 @@
...
+	return TRUE
+
+/obj/machinery/door/poddoor/welder_act(mob/living/user, obj/item/tool)
+	. = ..()
+	if (!panel_open)
+		return
+	if (deconstruction != BLASTDOOR_NEEDS_WIRES)
+		return
+	balloon_alert(user, "tearing apart...") //You're tearing me apart, Lisa!
+	if(tool.use_tool(src, user, 150, volume = 50))
```
Comment (Watermelon914) [inline code/game/machinery/doors/poddoor.dm:75]: change the time units in these to seconds (so 15 SECONDS here)

File: code/game/machinery/doors/poddoor.dm [hunk 3]
```diff
@@ -16,56 +15,70 @@
...
+		to_chat(user, span_notice("You change the ID to [id]."))
+		balloon_alert(user, "ID changed")
+
+/obj/machinery/door/poddoor/crowbar_act(mob/living/user, obj/item/tool)
+	. = ..()
+	if (!panel_open)
+		return
+	if (deconstruction != BLASTDOOR_FINISHED)
+		balloon_alert(user, "removing airlock electronics...")
+		if(tool.use_tool(src, user, 100, volume = 50))
```
Comment (Watermelon914) [inline code/game/machinery/doors/poddoor.dm:46]: same thing here and for the rest of the use_tool functions

**Pull request merged**